extensions/wikia/Venus/VenusController.class.php

```diff
@@ -81,6 +81,7 @@ private function setAds() {
 		$this->adPrefooterLeftBoxad = $this->app->renderView( 'Ad', 'Index', [ 'slotName' => 'PREFOOTER_LEFT_BOXAD', 'includeLabel' => true ] );
 		$this->adPrefooterRightBoxad = $this->app->renderView( 'Ad', 'Index', [ 'slotName' => 'PREFOOTER_RIGHT_BOXAD', 'includeLabel' => true ] );
 		$this->adsBottom = $this->app->renderView( 'Ad', 'Index', [ 'slotName' => 'GPT_FLUSH' ] );
+		$this->adsBottom = $this->app->renderView( 'Ad', 'Index', [ 'slotName' => 'TURTLE_FLUSH' ] );
```

Comment:
Let's not touch Venus at all. In fact I'll create a story for the CONCF team to clean this up.